release-doc-in-process/all_vf.TTL

```diff
@@ -322,17 +322,35 @@ vf:subject  a        owl:ObjectProperty ;
         rdfs:range   foaf:Agent ;
         rdfs:comment    "The subject of a relationship between 2 agents." .
 
+vf:object  a         owl:ObjectProperty ;
+        rdfs:domain  vf:AgentRelationship ;
+        rdfs:label   "object" ;
+        rdfs:range   foaf:Agent ;
+        rdfs:comment        "The object of a relationship between 2 agents." .
+
 vf:context  a        owl:ObjectProperty ;
         rdfs:domain  vf:AgentRelationship ;
         rdfs:label   "context" ;
         rdfs:range   foaf:Agent ;
         rdfs:comment        "The larger context of a relationship between 2 agents, used where the relationship is not relevant outside of that context." .
 
-vf:object  a         owl:ObjectProperty ;
-        rdfs:domain  vf:AgentRelationship ;
-        rdfs:label   "object" ;
-        rdfs:range   foaf:Agent ;
-        rdfs:comment        "The object of a relationship between 2 agents." .
+vf:scope  a        owl:ObjectProperty ;
```

Comment:
If we try to clearly capture **the relationship** that this property represents. Maybe *inScopeDefinedBy* would stand as clearer property name?